app/bin/tools/remove_package.dart

```diff
@@ -79,13 +80,13 @@ Future _deleteWithQuery<T>(Query query, {bool Function(T item) where}) async {
     if (shouldDelete) {
       deletes.add(m.key);
       if (deletes.length >= 500) {
-        await dbService.commit(deletes: deletes);
+        await _confirm(action: 'Deleting $T', deletes: deletes);
```

Comment:
I think this is too many times we have to type `y`.. I was just thinking a single `y/n` before it started all the deletes.

```
$ dart remove_package.dart remove mypkg
Deleting package: mypkg
Versions: ....
Are you sure you want to do this (y/n).
```

Just to catch cases where you type something wrong on the command line... Similar to how you have to confirm when publishing a package. Reviewing every single transaction is probably too much, especially since we can't do it in a single transaction.